recipes/protobuf/all/conanfile.py

```diff
@@ -199,9 +201,10 @@ def package_info(self):
         self.cpp_info.components["protoc"].name = "protoc"
         self.cpp_info.components["protoc"].requires.extend(["libprotoc", "libprotobuf"])
 
-        bindir = os.path.join(self.package_folder, "bin")
-        self.output.info("Appending PATH environment variable: {}".format(bindir))
-        self.env_info.PATH.append(bindir)
+        if not tools.cross_building(self.settings):
+            bindir = os.path.join(self.package_folder, "bin")
+            self.output.info("Appending PATH environment variable: {}".format(bindir))
+            self.env_info.PATH.append(bindir)
```

Comment:
```suggestion
        if not tools.cross_building(self.settings):
            bindir = os.path.join(self.package_folder, "bin")
            self.output.info("Appending PATH environment variable: {}".format(bindir))
            self.env_info.PATH.append(bindir)
```

What problem does it fix? If you are using two profiles it should not be a problem.

Reply:
I thought this was necessary to make sure the correct protoc was found but I just tried to reproduce the problem and it seems to work without this change. I will remove it and hope it's not a random fluke :)